app/src/main/java/abhishek/pathak/mockwithai/view/screens/UserAccountScreen.kt

```diff
@@ -1,26 +1,126 @@
 package abhishek.pathak.mockwithai.view.screens
 
+import abhishek.pathak.mockwithai.ui.theme.*
+import androidx.compose.foundation.Image
 import androidx.compose.foundation.background
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.material3.Text
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.GridCells
+import androidx.compose.foundation.lazy.LazyVerticalGrid
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.AccountBox
+import androidx.compose.material3.*
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
 
 @Composable
 fun UserAccountScreen() {
+    ScaffoldUI { paddingValues ->
+        Box(modifier = Modifier.padding(paddingValues)) {
+            Column(modifier = Modifier.fillMaxSize()) {
+                ProfileHeader()
+                MenuBar()
+                ImageGallery()
+            }
+        }
+    }
+}
 
-    Box(modifier = Modifier
-        .fillMaxSize()
-        .background(Color.Blue)) {
-        Text(text = "User Account Screen",
-            textAlign = TextAlign.Center,
+@Composable
+fun ProfileHeader() {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(dp_16)
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.profile_picture),
+            contentDescription = "Profile Picture",
```

Comment:
Content description should always explain about what it is? 
This is profile picture